hamocc/mo_read_ndep.F90

```diff
@@ -269,7 +252,7 @@ subroutine get_ndep(kpie,kpje,kbnd,kplyear,kplmon,omask,ndep,patmnhxdep,patmnoyd
             ndep(i,j,idepnoy) = noydepread(i,j)
             ndep(i,j,idepnhx) = nhxdepread(i,j)
           else
-            ndep(i,j,idepnoy) = ndepread(i,j)
+            ndep(i,j,idepnoy) = noydepread(i,j) + nhxdepread(i,j)
```

Comment:
I think the `ndep = 0.0` in line 203 should be moved up to the start of the routine (could then also be removed in line 189). Currently it is only set to zero for `use_nuopc_ndep=.true.`

Reply:
Done.